pkg/base/tools.go

```diff
@@ -34,6 +73,9 @@ func IsValidType(t string) bool {
 	case "bool":
 	case "timestamp":
 	default:
+		if strings.HasPrefix(t, "date-timestamp") && len(strings.Split(t, ":")) == 2 {
```

Comment:
why add this `date-timestamp` type ? this data type is not supported in nebula graph. The date type will be added in next version. And now you can use `timestamp` or `string` type to store your date data.